src/main/java/Car.java

```diff
@@ -0,0 +1,23 @@
+/* A sample class that implements the given interfaces.
+ */
+public class Car implements Drivable, Domesticatable, Tradable {
+    public int maxSpeed;
```

Comment:
Consider making this field private, as there is already an accessor method defined.